### DISCUSSION TITLE
Special characters

### DISCUSSION THREAD

**Discussion by WeirdGamerAidan** [TownyAdvanced/Towny#5928] [Q&A] [answered]
What special characters are allowed in town names? I know putting an underscore ( _ ) puts a space in some scenarios, and I'm mainly wondering about commas ( , ).

**Comment by LlmDl** [comment, marked as answer]
Spaces aren't allowed, but putting a _ renders as a space most of the time.
The [How Towny Works wiki page has a section about regex](https://github.com/TownyAdvanced/Towny/wiki/How-Towny-Works#towny-regex-settings) which determines what you can use for characters in many named objects. It wouldn't be a good idea to start allowing punctuation.

**Reply by WeirdGamerAidan** [reply]
> Spaces aren't allowed, but putting a _ renders as a space most of the time.
I said that in my original message
 > The How Towny Works wiki page has a section about regex which determines what you can use for characters in many named objects. It wouldn't be a good idea to start allowing punctuation.
Thanks, I'll take note of that

**Reply by WeirdGamerAidan** [reply]
Tf i thought it would only put part of it as a quote
Whatever idk how quoting works
